lib/contact_manager.rb

```diff
@@ -8,4 +8,9 @@ def initialize
     @contact_list = ContactList.new
   end
 
+  def display_contacts
+    @list.each do |key, value|
```

Comment:
Same comment as above re. better naming for `key` and `value`. Of course these make sense, but it would make much more sense if in the future you'd look at your code and see names like `last_name`, `last_name_entered` or something like that

Reply:
well, this one is still blurry for me 😅

Reply:
You mean my comment or the syntax? :)